[Documentation] Document the configuration of Project JDK and Maven JDK in IntelliJ (#10371)
apache/pulsar: README.md

### Intellij

#### Configure Project JDK to Java 8 (1.8) JDK

1. Open Project Settings dialog by going to  

>    `File -> Project Structure -> Project Settings -> Project`.

Reviewer: In technical writing, normally, we use "bold" for buttons instead of "code style" (``).
Please check all occurencies.

> 2. Click on the JDK version drop down and select `Download JDK...` or choose an existing recent Java 8 (1.8) JDK version.

Reviewer: ```suggestion
2. Select the JDK version.
    
    From the JDK version drop-down list, select **Download JDK...** or choose an existing recent Java 8 (1.8) JDK version.
```

> 3. In the download dialog, select version `1.8` and vendor `AdoptOpenJDK (Hotspot)` for the JDK to download.

Reviewer 2: I am not sure we should suggest a particular distribution.
I like AdoptOpenJDK but it is not required

Author: I have rephrased the wording to address your comment. PTAL

> 3. In the download dialog, select version `1.8`. You can pick from many vendors. Unless you have a specific preference, choose  `AdoptOpenJDK (Hotspot)`.

Reviewer 3: ```suggestion
3. In the download dialog, select version `1.8`. You can pick a version from many vendors. Unless you have a specific preference, choose `AdoptOpenJDK (Hotspot)`.
```
